ui/src/pages/frontsite/Documents/DocumentsDetails/DocumentsDetails.js

```diff
@@ -63,7 +63,14 @@ export default class DocumentsDetails extends Component {
   ];
 
   render() {
-    const { isLoading, error, documentDetails } = this.props;
+    const { isLoading, hasError, error, documentDetails } = this.props;
```

Comment:
@ntarocco, @kprzerwa, @zzacharo, instead of explicitly checking in every component maybe we can have a HOC that will handle the errors and their relevant components

Reply:
yep, maybe for @kprzerwa refactoring in the new repo, since this affects and it is applicable only to the main details pages components routes.